rhalphalib/model.py

```diff
@@ -192,18 +192,19 @@ def getObservation(self):
         Return the current observation set for this Channel as plain numpy array
         If it is non-poisson, it will be returned as a tuple (sumw, sumw2)
         '''
+        _mask_val = float(0)
```

Comment:
Would this make more sense as a class member? Also what's the use case for non-zero?

Reply:
Done. Use case was for debugging, where combine seemed to complain about 0 bins vs small epsilon.